baybe/serialization/utils.py

```diff
@@ -0,0 +1,19 @@
+"""A collection of serialization utilities."""
+
+from typing import Any
+
+import pandas as pd
+
+
+def serialize_dataframe(df: pd.DataFrame, /) -> Any:
```

Comment:
No, it's really just to encapsulate the logic gives you the required serialization string for the JSON config. Although we both agree that the current binary approach is not yet optimal, we should (at least temporarily) expose some public method that takes care of the conversion, I think. Otherwise, I'd need to tell my colleagues who use the config "to get your dataframe string, import that one custom converter, bla bla ..." <--- not so nice. Are you OK with this or do you have a better idea?

Reply:
ok argument makes sense --> provide a utility for people